Bottom: cd777575545a8e99abcfdb43a3a37d2eb8adbe87
Top:    a70e6e55e624bfa6c913daf4b55334c5a9be32b7
Author: Ian Kent <ikent@redhat.com>
Date:   2013-09-23 13:26:04 +0800

Refresh of debug.patch

---

diff --git a/aclocal.m4 b/aclocal.m4
index 637a775..7c7e405 100644
--- a/aclocal.m4
+++ b/aclocal.m4
@@ -428,6 +428,8 @@ if test "$af_have_libtirpc" = "yes"; then
     TIRPCLIB="-ltirpc"
 fi
 
+AC_CHECK_FUNCS([getrpcbyname])
+
 # restore flags
 CFLAGS="$af_check_libtirpc_save_cflags"
 LDFLAGS="$af_check_libtirpc_save_ldflags"
